VultisigApp/VultisigApp/Extensions/TextField.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  TextField.swift
+//  VultisigApp
+//
+//  Created by Enrique Souza Soares on 30/06/24.
+//
+// This file extension is used to limit the number of characters that can be entered in a TextField.
+
+import Foundation
+import SwiftUI
+import Combine
+
+#if canImport(UIKit)
+import UIKit
+
+private var maxLengthKey: UInt8 = 0
+
+extension UITextField {
+    @IBInspectable var maxLength: Int {
```

Comment:
IBInspectable can be removed, we're do not use interface builder

Reply:
It is used to set max length to both macOs and IOs.

Reply:
I think what @flypaper0 means is `@IBInspectable` is used for interface builder , we are not using that , so the property wrapper should be removed

Reply:
Hum... let me check

Reply:
He is right, we don't need that the extension resolves that. Thanks @flypaper0